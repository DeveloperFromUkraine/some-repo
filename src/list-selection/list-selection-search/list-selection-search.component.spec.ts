import { ComponentFixture, TestBed } from "@angular/core/testing";
import { ListSelectionSearchComponent } from "./index";
import { ComponentTest } from "../../../test/test-bed/component";
import { ReactiveFormsModule } from "@angular/forms";

describe('List Selection Search', () => {
    let fixture: ComponentFixture<ListSelectionSearchComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([ReactiveFormsModule], [ListSelectionSearchComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(ListSelectionSearchComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    });
});