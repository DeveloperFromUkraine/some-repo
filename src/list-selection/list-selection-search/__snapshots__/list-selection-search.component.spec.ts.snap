// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListSelectionSearchComponent should match snapshot 1`] = `
<ign-list-selection-search
  filterInput$={[Function AnonymousSubject]}
  searchCtrl={[Function FormControl]}
>
  <div
    class="container"
  >
    <i
      class="material-icons search-icon"
    >
      search
    </i>
    <input
      class="input"
      type="text"
    />
  </div>
</ign-list-selection-search>
`;
